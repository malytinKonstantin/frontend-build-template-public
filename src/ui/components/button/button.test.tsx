import React from 'react'
import Button from './button'
import renderer from 'react-test-renderer'

it('button renders correctly', () => {
  const tree = renderer.create(<Button type="primary">Facebook</Button>).toJSON()
  expect(tree).toMatchSnapshot()
})
