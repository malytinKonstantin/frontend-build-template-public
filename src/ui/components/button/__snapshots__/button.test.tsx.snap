// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button renders correctly 1`] = `
<button
  className="ant-btn ant-btn-primary"
  onClick={[Function]}
  type="button"
>
  <span>
    Facebook
  </span>
</button>
`;
